sponsorship/src/main/java/com/kyriba/conference/sponsorship/domain/EmailMessage.java

```diff
@@ -0,0 +1,47 @@
+package com.kyriba.conference.sponsorship.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class EmailMessage
+{
+  private List<Recipient> recipients;
+  private String body;
+  private String subject;
+  private Set<Options> emailMessageOptions;
+
+
+  @SuppressWarnings("unused")
+  public enum Options
+  {
+    HIGH_PRIORITY, HIDE_ADDRESS
+  }
+
+
+  public static EmailMessage createForSponsor(Sponsor sponsor)
```

Comment:
Why static methods? This logic must be in Sponsor class because it incapsulates internal state.